tools/RAiDER/delayFcns.py

```diff
@@ -98,14 +98,14 @@ def getInterpolators(wm_file, kind='pointwise', shared=False):
             ys_wm = np.array(ds.variables['y'][:])
             zs_wm = np.array(ds.variables['z'][:])
             wet = ds.variables['wet_total' if kind=='total' else 'wet'][:]
-            hydro = ds.variables['hydro_total' if kind=='total' else 'wet'][:]
+            hydro = ds.variables['hydro_total' if kind=='total' else 'hydro'][:]
     else:
         ds = wm_file
         xs_wm = np.array(ds.variables['x'][:])
         ys_wm = np.array(ds.variables['y'][:])
         zs_wm = np.array(ds.variables['z'][:])
         wet = ds.variables['wet_total' if kind=='total' else 'wet'][:]
-        hydro = ds.variables['hydro_total' if kind=='total' else 'wet'][:]
```

Comment:
Bug correction?

Reply:
It's there, so not sure why it shows deleted here.

Reply:
the one that stayed has else 'hydro' not wet.